chrome/content/zotero/xpcom/style.js

```diff
@@ -81,9 +81,41 @@ Zotero.Styles = new function() {
 		
 		// hidden dir
 		dir.append("hidden");
-		if(dir.exists()) i += _readStylesFromDirectory(dir, true);
+		if (dir.exists()) i += _readStylesFromDirectory(dir, true);
 		
 		Zotero.debug("Cached "+i+" styles in "+((new Date()).getTime() - start)+" ms");
+		
+		// transfer and reset "export.bibliographyLocale" pref value
+		var bibliographyLocale = '';
+		bibliographyLocale = Zotero.Prefs.get("export.bibliographyLocale");
+		if (bibliographyLocale) {
+			Zotero.Prefs.set("export.lastLocale", bibliographyLocale);
+			Zotero.Prefs.set("export.quickCopy.locale", bibliographyLocale);
+			Zotero.Prefs.clear("export.bibliographyLocale");
+		}
+		
+		// load available CSL locales
+		var localeFile = {};
+		var locales = {};
+		var primaryDialects = {};
+		var localesLocation = "chrome://zotero/content/locale/csl/locales.json";
+		localeFile = JSON.parse(Zotero.File.getContentsFromURL(localesLocation));
+		
+		for (let dialect in localeFile["primary-dialects"]) {
+			if (localeFile["primary-dialects"][dialect] !== undefined) {
+				primaryDialects[dialect] = localeFile["primary-dialects"][dialect];
+			}
+		}
+		
+		// only keep localized language name
+		for (let locale in localeFile["language-names"]) {
+			if (localeFile["language-names"][locale] !== undefined) {
```

Comment:
Taken care of in https://github.com/rmzelle/zotero/commit/6dacf7b5e8950c0fa09ce11246f9c1f66f9fa5e0